<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calculator_0827</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="4" column="0">
     <widget class="QPushButton" name="tanBtn">
      <property name="text">
       <string>tan</string>
      </property>
     </widget>
    </item>
    <item row="5" column="3">
     <widget class="QPushButton" name="digitBtn9">
      <property name="text">
       <string>9</string>
      </property>
     </widget>
    </item>
    <item row="6" column="2">
     <widget class="QPushButton" name="digitBtn0">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QPushButton" name="powBtn">
      <property name="text">
       <string>^</string>
      </property>
     </widget>
    </item>
    <item row="4" column="4">
     <widget class="QPushButton" name="substactionBtn">
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="digitBtn4">
      <property name="text">
       <string>4</string>
      </property>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="QPushButton" name="equalBtn">
      <property name="text">
       <string>=</string>
      </property>
     </widget>
    </item>
    <item row="5" column="4">
     <widget class="QPushButton" name="multiplicationBtn">
      <property name="text">
       <string>*</string>
      </property>
     </widget>
    </item>
    <item row="6" column="3">
     <widget class="QPushButton" name="signBtn">
      <property name="text">
       <string>+/-</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QPushButton" name="pointBtn">
      <property name="text">
       <string>.</string>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QPushButton" name="digitBtn6">
      <property name="text">
       <string>6</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="digitBtn2">
      <property name="text">
       <string>2</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QPushButton" name="digitBtn3">
      <property name="text">
       <string>3</string>
      </property>
     </widget>
    </item>
    <item row="6" column="4">
     <widget class="QPushButton" name="divisionBtn">
      <property name="text">
       <string>/</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="digitBtn1">
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="cosBtn">
      <property name="text">
       <string>cos</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QPushButton" name="digitBtn7">
      <property name="text">
       <string>7</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QPushButton" name="lnBtn">
      <property name="text">
       <string>ln</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QPushButton" name="digitBtn8">
      <property name="text">
       <string>8</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QPushButton" name="digitBtn5">
      <property name="text">
       <string>5</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QPushButton" name="lgBtn">
      <property name="text">
       <string>lg</string>
      </property>
     </widget>
    </item>
    <item row="3" column="4">
     <widget class="QPushButton" name="addBtn">
      <property name="text">
       <string>+</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="rightBracketBtn">
      <property name="text">
       <string>)</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="leftBracketBtn">
      <property name="text">
       <string>(</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="5">
     <widget class="QTextEdit" name="textEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="percentBtn">
      <property name="text">
       <string>%</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="sinBtn">
      <property name="text">
       <string>sin</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="2">
     <widget class="QPushButton" name="clearBtn">
      <property name="text">
       <string>clear</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3" colspan="2">
     <widget class="QPushButton" name="ClearAllBtn">
      <property name="text">
       <string>ClearAll</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>450</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>菜单</string>
    </property>
    <addaction name="action_V"/>
    <addaction name="action"/>
    <addaction name="action_2"/>
    <addaction name="action_3"/>
    <addaction name="action_4"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action"/>
   <addaction name="action_2"/>
   <addaction name="action_3"/>
   <addaction name="action_4"/>
   <addaction name="action_5"/>
   <addaction name="action_6"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_V">
   <property name="text">
    <string>粘贴 &amp; Ctrl+V</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>历史记录</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>清除记录</string>
   </property>
  </action>
  <action name="action_3">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>科学计算器</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>二进制</string>
   </property>
   <property name="toolTip">
    <string>二进制</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>日期</string>
   </property>
   <property name="toolTip">
    <string>日期</string>
   </property>
   <property name="menuRole">
    <enum>QAction::TextHeuristicRole</enum>
   </property>
  </action>
  <action name="action_6">
   <property name="text">
    <string>程序员</string>
   </property>
   <property name="toolTip">
    <string>程序员</string>
   </property>
   <property name="menuRole">
    <enum>QAction::TextHeuristicRole</enum>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
